zk_ee/src/memory/slice_vec.rs

```diff
@@ -78,6 +81,8 @@ impl<T: Clone> SliceVec<'_, T> {
             }
```

Comment:
I can not comment above this line, but it's related to drop loop above. Hopefully compiler can eliminate a loop completely if drop is noop, but it may be easier to wrap it into "if core::mem::needs_drop::<T>()" that is usually much better eliminated

Reply:
Original Rust doc suggest that you can drop_in_place a slice directly, that would do the same check

Reply:
For the current use cases this doesn't matter but dropping a slice is a bit less code, so sounds good to me.

Reply:
Replaced with `drop_in_place`, binary is the same

Reply:
Thanks! Added cast, hope no more stupid mistakes

Reply:
I'm going to merge it. @joonazan please check this part later